src/Pages/SingleVideo/SingleVideo.jsx

```diff
@@ -1,35 +1,75 @@
+  
+    const { videoId } = useParams();
+    //For Liked Video
+    const {
+        saveLikedVideo,
+        state: { likedVideo },
+    } = useLikesContext();
 
     // Now match params video ID from existing videos data
-    const isVideo = getFilteredVideo?.find((eachVideo) => eachVideo._id === videoId)
-   
-    return getFilteredVideo && (<div className="single-videos-container main-container" style={{ marginTop: "5rem" }}>
-    <div className="single-video-wrapper">
-        <div className="video-player">
-            <VideoIframe videoId={isVideo?._id} />
-            <div className="video-info">
-                <h1 className="space-between">{isVideo?.title} <span className="text-xm grey-text center"><FaIcons.FaEye className="view-icon"/> {isVideo?.view}</span></h1>
-                <h3 className="creator-name">{isVideo?.creator}<FaIcons.FaCheckCircle className="icons verified-icon" /></h3>
-                <ul>
-                    <li className="text-lg"><FaIcons.FaThumbsUp className="icons sidebar-icons" /> Likes</li>
-                    <li className="text-lg"><FaIcons.FaArchive className="icons sidebar-icons" /> Watch Later</li>
-                </ul>
-                <p className="text-lg video-description video-full-description"> <strong>Description:</strong> {isVideo?.description}</p>
-            </div>
-        </div>
-    </div>
-</div> )
+    const isVideo = getFilteredVideo?.find(
+        (eachVideo) => eachVideo._id === videoId
+    );
+    // liked video
+    const likeVideoHandler = () => {
+        saveLikedVideo(isVideo);
+    };
+
+    return (
+        getFilteredVideo && (
+            <div
+                className="single-videos-container main-container"
+                style={{ marginTop: "5rem" }}
+            >
+                <div className="single-video-wrapper">
+                    <div className="video-player">
+                        <VideoIframe videoId={isVideo?._id} />
+                        <div className="video-info">
+                            <h1 className="space-between">
+                                {isVideo?.title}{" "}
+                                <span className="text-xm grey-text center">
+                                    <FaIcons.FaEye className="view-icon" /> {isVideo?.view}
+                                </span>
+                            </h1>
+                            <h3 className="creator-name">
+                                {isVideo?.creator}
+                                <FaIcons.FaCheckCircle className="icons verified-icon" />
+                            </h3>
+                            <ul>
+                                <li
+                                    className="text-lg btn-likes"
+                                    onClick={() => likeVideoHandler()}
+                                >
+                                    <FaIcons.FaThumbsUp className="icons sidebar-icons" /> Like
+                                </li>
```

Comment:
By performing a check if the video is already liked, you can add a toggle here and give an option to remove the like.

Reply:
It Done. Thank you